manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -89,31 +88,43 @@ Then the source of the add-on package will be checked out into the ``src`` folde
 
 .. note::
 
-    You can also add Products.PDBDebugMode and plone.reload to your development-tools:
+    You can also add some development-tools like ``Products.PDBDebugMode`` and ``plone.reload`` to your buildout.
+    For some you currently need a source checkout though.
 
```

Comment:
Add a link to docs about source checkout

Reply:
What is the link? I don't know the correct one.

Reply:
This is a good question, I would go with https://training.plone.org/5/mastering-plone/buildout_1.html#buildout-i
Even it is not perfect it gives useful info.
Longtime Plone devs may not need it, but if you are new, this may help you.

Reply:
I'm also unclear about whether "For some" refers to buildouts or people doing buildouts. Can you clarify?

Reply:
As I read and understand it that "For some" is meaning "For some development tools"
But this is only my guess :)

Reply:
Added link for source checkout in 0acc5b7

Reply:
Improved wording slightly in 8ef23c8. It's still klunky.